p/scripts/main.js

```diff
@@ -846,6 +846,9 @@ function init_column_categories() {
 				a.href = '#dropdown-' + id;
 				div.querySelector('.dropdown-target').id = 'dropdown-' + id;
 				div.insertAdjacentHTML('beforeend', template);
+				if (feed_web.length < 1) {
```

Comment:
What was the reason for having this logic client side in JavaScript?
Could be put server side in PHP in `app/layout/aside_feed.phtml`, don't you think?

Reply:
Sorry, wrong comment. It is in a template, so all good